kaldb/src/main/java/com/slack/kaldb/logstore/search/KaldbDistributedQueryService.java

```diff
@@ -24,6 +27,17 @@
 
   public static List<String> servers = new ArrayList<>();
 
+  private final LoadingCache<String, KaldbServiceGrpc.KaldbServiceFutureStub> stubLoadingCache =
```

Comment:
Does this map need data from ZK? If so, we need to initialize this field via a method.

Reply:
This does not - it just provides a Function that is invoked when the loading cache has a miss. I refactored this slightly to make this clearer by breaking out the GRPC client initialization into a separate method.